linux_os/guide/services/cron_and_at/file_groupowner_cron_daily/tests/other_group.fail.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# profiles = xccdf_org.ssgproject.content_profile_C2S
+GROUP=ssgttgroup
+
```

Comment:
The test scenario fails for me.

```
[jcerny@thinkpad scap-security-guide{cis_cron_perms}]$ python3 tests/test_suite.py rule  --libvirt qemu:/
//system ssgts_rhel7 --datastream build/ssg-rhel7-ds-1.2.xml file_groupowner_cron_daily
Setting console output to log level INFO
INFO - The DataStream contains 2 Benchmarks
INFO - 0 - scap_org.open-scap_cref_ssg-rhel7-xccdf-1.2.xml
INFO - 1 - scap_org.open-scap_cref_ssg-rhel7-pcidss-xccdf-1.2.xml
INFO - Selected Benchmark is 0
INFO - To select a different Benchmark, use --xccdf-id-number option.
INFO - The base image option has not been specified, choosing libvirt-based test environment.
INFO - Logging into /home/jcerny/scap-security-guide/logs/rule-custom-2019-09-23-1517/test_suite.log
INFO - xccdf_org.ssgproject.content_rule_file_groupowner_cron_daily
ERROR - Script is_root.pass.sh using profile xccdf_org.ssgproject.content_profile_C2S found issue:
ERROR - Rule evaluation resulted in fail, instead of expected pass during initial stage 
ERROR - The initial scan failed for rule 'xccdf_org.ssgproject.content_rule_file_groupowner_cron_daily'.
INFO - Script other_group.fail.sh using profile xccdf_org.ssgproject.content_profile_C2S OK
ERROR - Rule evaluation resulted in error, instead of expected fixed during remediation stage 
ERROR - The remediation failed for rule 'xccdf_org.ssgproject.content_rule_file_groupowner_cron_daily'.
```

I suspect that it's related to the CSV file which leads to generating a different check than we want.